example_client/grpc_latency.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
Did you try yapf?
Also I checked with flake8 style check and this is its report:
```
grpc_latency.py|20 col 1 error| 'classes' imported but unused [F401]
grpc_latency.py|24 col 1 error| 'tensorflow.make_ndarray' imported but unused [F401]
grpc_latency.py|44 col 1 error| continuation line under-indented for visual indent [E128]
grpc_latency.py|51 col 35 warning| trailing whitespace [W291]
grpc_latency.py|55 col 36 warning| trailing whitespace [W291]
grpc_latency.py|56 col 38 warning| trailing whitespace [W291]
grpc_latency.py|58 col 38 warning| trailing whitespace [W291]
grpc_latency.py|62 col 28 warning| trailing whitespace [W291]
grpc_latency.py|76 col 58 warning| trailing whitespace [W291]
grpc_latency.py|77 col 42 warning| trailing whitespace [W291]
grpc_latency.py|98 col 35 warning| trailing whitespace [W291]
grpc_latency.py|108 col 54 error| missing whitespace after ',' [E231]
grpc_latency.py|114 col 28 warning| trailing whitespace [W291]
grpc_latency.py|115 col 36 warning| trailing whitespace [W291]
grpc_latency.py|118 col 121 error| line too long (133 > 120 characters) [E501]
```